pulsar-broker-common/src/main/java/org/apache/pulsar/broker/resources/NamespaceResources.java

```diff
@@ -295,6 +296,75 @@ public CompletableFuture<Void> clearPartitionedTopicTenantAsync(String tenant) {
             final String partitionedTopicPath = joinPath(PARTITIONED_TOPIC_PATH, tenant);
             return deleteIfExistsAsync(partitionedTopicPath);
         }
+
+        public CompletableFuture<Void> markPartitionedTopicDeletedAsync(TopicName tn) {
+            if (tn.isPartitioned()) {
+                return CompletableFuture.completedFuture(null);
+            }
+            log.debug("markPartitionedTopicDeletedAsync {}", tn);
```

Comment:
nit: across the code base we use "if log.isDebugEnabled" guards.
we are not in a hot path but that's the style used by the project

Reply:
I don't think it is needed there. "if log.isDebugEnabled" guards against unnecessary boxing of primitives, tn is an Object already.